dev-containers-vscode/README.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started with Development Containers in VS Code
+
+## Module Source link
```

Comment:
looking great! Could you comb through for /en-us/ - and remove it, so we don't fix the links to US English

Reply:
done!